packages/bpk-component-text/readme.md

```diff
@@ -0,0 +1,61 @@
+# bpk-component-text
+
+> Backpack text component.
+
+## Installation
+
+```sh
+npm install bpk-component-text --save-dev
+```
+
+## Usage
+
+```javascript
+import React from 'react';
+import BpkText from 'bpk-component-text';
+
+export default () => (
+  <BpkText tagName="h1" textStyle="xl">My Heading</BpkText>
+);
+```
+
+When using the same style in many places repeating the `textStyle` and `tagName` props can become tedious in this case you can use `withTextStyle`
+
+```javascript
+import React from 'react';
+import BpkText, { withTextStyle } from 'bpk-component-text';
+
+const LargeParagraph = withTextStyle(BpkText, 'lg', 'p');
+const TinySpan = withTextStyle(BpkText, 'xs', 'span');
+
+export default () => (
+  <div>
+	  <LargeParagraph>
+	      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+	      Pellentesque imperdiet lobortis tellus, non rhoncus erat tincidunt id.
+	      Pellentesque consectetur, dolor nec vulputate vehicula, ex metus mattis ante,
+	      non dictum mi ante eu arcu.
+	  </LargeParagraph>
+  	  <LargeParagraph>
+	      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+	      Pellentesque imperdiet lobortis tellus, non rhoncus erat tincidunt id.
+	      Pellentesque consectetur, dolor nec vulputate vehicula, ex metus mattis ante,
+	      non dictum mi ante eu arcu.
+	  </LargeParagraph>
+  	  <TinySpan>
+  		Side effects of Backpack include euphoria, happiness, and increased develpoment velocity.
+      </TinySpan> 
+  </div>
+);
+```
+
```

Comment:
Yeah this is just down to my markdown editor not being great at displaying source level whitespace